### PR TITLE
Delete post

### PR DESCRIPTION
**Pull request by lucianmot**
Deletus

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -4,11 +4,21 @@ def new
     @post = current_user.posts.new
   end
 
+  # def find
```
Comment (rachjgriff) [inline app/controllers/posts_controller.rb]: hmmm is this for something in the future?

File: spec/features/user_can_delete_a_post_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.feature "Delete a post", type: :feature do
+  scenario "when you click on delete, post is removed from your page" do
+    sign_up
+    click_link "View Your Posts"
+    create_a_post("I love monsieur Travis")
```
Comment (LucyMHall) [inline spec/features/user_can_delete_a_post_spec.rb]: I suspect that this is not entirely true

Comment (lucianmot) [reply spec/features/user_can_delete_a_post_spec.rb]: we will never ever truly know

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (Danielandro) [reply app/controllers/posts_controller.rb]: Yes it is. Possibly for the edit functionallity

**Pull request merged**